fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthConfiguration.java

```diff
@@ -33,66 +33,74 @@
 /**
  * Configuration of metadata about a fixed width values datacontext.
  */
-public final class FixedWidthConfiguration extends BaseObject implements
-		Serializable {
+public final class FixedWidthConfiguration extends BaseObject implements Serializable {
 
-	private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-	public static final int NO_COLUMN_NAME_LINE = 0;
-	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
+    public static final int NO_COLUMN_NAME_LINE = 0;
+    public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
-	private final String encoding;
-	private final int fixedValueWidth;
-	private final int[] valueWidths;
-	private final int columnNameLineNumber;
-	private final boolean failOnInconsistentLineWidth;
-	private final ColumnNamingStrategy columnNamingStrategy;
+    private final String encoding;
+    private final int fixedValueWidth;
+    private final int[] valueWidths;
+    private final int columnNameLineNumber;
+    private final boolean failOnInconsistentLineWidth;
+    private final boolean headerPresent;

```

Comment:
Yes, let's give it a better name. I am not too much into "padding" since some padding is actually in all rows. Basically this "binary header" contains some additional data about the file, so what about `metadataPresent`? Would that be clear to the users?